rfcs/0043-rfcsc-rotation.md

```diff
@@ -0,0 +1,102 @@
+---
+feature: rfcsc-rotation
+start-date: 2019-04-24
+author: Robin Gloster <mail@glob.in>, Simon Lackerbauer <mail@ciil.io>
+related-issues: 36
+---
+
+# Summary
+[summary]: #summary
+
+Each RFC Steering Committee (RFCSC) unanimously elects the succeeding one at
+their first meeting in December from an open list of nominees.
+
+# Motivation
+[motivation]: #motivation
+
+With the implementation of [RFC
+36](https://github.com/NixOS/rfcs/blob/master/rfcs/0036-rfc-process-team-amendment.md)
+the RFC Steering Committee has been established. Among others, future work for
+that RFC included a definition of how members to the Committee are to be chosen
+or removed. This RFC provides mechanisms through which membership in the
+Steering Committee can be established or ended, so as not to make membership a
+mandate for life, allowing a healthy rotation of members, and offering several
+procedures to always keep the Steering Committee capable of making decisions
+such that the RFC process can keep running smoothly at all times.
```

Comment:
Run-on sentence! :rotating_light: The "mandate for life" and "healthy rotation" bits are also just restating that the committee will change and that that is in some vague way good. Here's an attempt at rephrasing with a bit more structure and justification:

> This RFC provides mechanisms for beginning and ending Steering Committee membership. The purpose of this is to ensure that the committee:
> - continues to consist of well-informed and motivated members;
> - continues to represent the evolving community;
> - remains active and keeps the RFC process running.